core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -408,7 +412,11 @@ private ClusterCacheStatus initCacheStatusIfAbsent(String cacheName) {
          } else {
             availabilityStrategy = new PreferAvailabilityStrategy(eventLogManager);
          }
-         return new ClusterCacheStatus(cacheName, availabilityStrategy, this, transport);
+         Optional<GlobalStateManager> globalStateManager = cacheManager.getGlobalComponentRegistry().getOptionalComponent(GlobalStateManager.class);
```

Comment:
I didn't want to waste a field for storing a component which I only need once to retrieve the persistent state